website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Connecting your GitLab Account"
+id: "connecting-gitlab"
+sidebar_label: "Connecting your GitLab Account"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, let us know at sales@getdbt.com.
+
+:::
+
+## Overview
+
+Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
+- trigger CI builds when Pull Requests are opened in GitLab!
+- log into dbt Cloud via oauth through GitLab!
+- add new repos to Cloud in a single click!
+- and more!
+
+## Linking dbt Cloud to your GitLab account
+
+To link your dbt Cloud account to your GitLab account, navigate to your Account Settings and click the Integrations tab. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/gitlab_navigation_setup.gif" title="Like so!"/>
+
+Here you can link your GitLab account to dbt Cloud. 
+
+You'll need to set up an application at GitLab in order to make it official - [they have a guide for that here](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application-through-the-profile)! For your dbt Cloud application, you'll want to check the `api` scope level.
+
+Once you've gotten your Application ID and Secret from GitLab, you can click the Edit button in your Integrations tab, and copy-paste those values into the form.
```

Comment:
Can we also include information about how to set the `gitlab instance` field? I bet this is going to be a common cause of misconfiguration - the `gitlab instance` field should contain the hostname of the instance where gitlab is running, eg. `https://gitlab.com` or `https://gitlab.example.com`. If a user supplied `https://gitlab.com/` (with a trailing slash) or `gitlab.com` (without a protocol) then the integration will not work correctly

Reply:
Great point here - It would be great if we could make smarter forms for this sort of thing: it feels like there's probably a package for this already, right?

Reply:
hmmmm, do you mean better forms on the docs site? What kind of thing are you picturing?